src/firefox/lib/glue.js

```diff
@@ -170,6 +180,58 @@ function setUpConnection(freedom, panel, button) {
       });
     }
   });
+
+
+  // Check if user already has a tab open to the uProxy install page.
+  for (var tab of tabs) {
+    if (isInstallPage(tab.url)) {
```

Comment:
I'd say that, for the sake of consistency, we should probably do this for anything that matches the url pattern normally used for attaching the content scripts.